index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>survey form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+    <form id="survey-form">
+      <label for="name" id="name-label">Name</label>
+      <input type="text" id="name" placeholder="Enter Your Name" />
+      <label for="email" id="email-label">Email</label>
+      <input type="email" id="email" placeholder="Enter Your Email" />
+      <label for="number" id="number-label">Number</label>
+      <input
+        min="5"
+        max="15"
+        type="number"
+        id="number"
+        placeholder="Enter Your Number"
+      />
+      <label for="languages">Choose a language</label>
```

Comment:
I believe you should add a "required" to your forms so everything should have to be filled :thinking:

Reply:
- [optional] Awesome! But I think that you can frame it better` "for validation purpose"`. 💁‍♂️